scripts/test.sh

```diff
@@ -96,7 +97,7 @@ function main {
     # Dependencies required for testing
     go get -u github.com/kardianos/govendor
     govendor init
-    govendor fetch github.com/dgrijalva/jwt-go/^::gopkg.in/dgrijalva/jwt-go.v2
+    govendor fetch github.com/dgrijalva/jwt-go/^::gopkg.in/dgrijalva/jwt-go.v3
```

Comment:
this is not related to this PR, and also did you validate that this works? last time i tried it it did not work for some reason...